back-end/domain/status/StatusManager.js

```diff
@@ -71,6 +73,18 @@ class StatusManager {
         this.statuses = this.statuses.filter(status => !status.isOld());
     }
 
+    checkForOldStatuses() {
```

Comment:
This PR looks great! Awesome. 
I do feel the naming of this method is a bit off. Especially considering `removeOldStatuses()`, which only removes the statusses within the server, that's fair. But this function does more that just checking for old statuses. Have you considered `pushOldStatusRemoval()` OSLT?

Reply:
Done!